src/meta/src/telemetry.rs

```diff
@@ -14,99 +14,17 @@
 
 use std::sync::Arc;
 
-use anyhow::anyhow;
 use risingwave_common::config::MetaBackend;
 use risingwave_common::telemetry::report::{TelemetryInfoFetcher, TelemetryReportCreator};
 use risingwave_common::telemetry::{
     current_timestamp, SystemData, TelemetryNodeType, TelemetryReport, TelemetryReportBase,
 };
 use risingwave_pb::common::WorkerType;
 use serde::{Deserialize, Serialize};
-use uuid::Uuid;
 
 use crate::manager::ClusterManager;
-use crate::model::{MetadataModelError, MetadataModelResult};
-use crate::storage::{MetaStore, Snapshot};
-
-/// Column in meta store
-pub const TELEMETRY_CF: &str = "cf/telemetry";
-/// `telemetry` in bytes
-pub const TELEMETRY_KEY: &[u8] = &[74, 65, 0x6c, 65, 0x6d, 65, 74, 72, 79];
```

Comment:
The only concern is that clusters may lose the previous persistent `tracking_id` after upgradation. Not a big problem.